libcodechecker/orm_model.py

```diff
@@ -241,6 +206,10 @@ class Report(Base):
                                      initially="DEFERRED", ondelete='CASCADE'),
                           index=True)
     suppressed = Column(Boolean)
+    detection_status = Column(Enum('new',
+                                   'unresolved',
+                                   'resolved',
+                                   'reopened'))
```

Comment:
Yes, we keep `reopened`. Moreover in the latest implementation a report stays reopened unit it becomes resolved again.